test/battle/move_effect/stockpile.c

```diff
@@ -48,19 +48,19 @@ SINGLE_BATTLE_TEST("Spit Up and Swallow don't work if used without Stockpile")
         TURN { MOVE(player, move); }
     } SCENE {
         NOT ANIMATION(ANIM_TYPE_MOVE, move, player);
-        if (move == MOVE_SWALLOW)
+        if (move == MOVE_SWALLOW) {
             MESSAGE("But it failed to swallow a thing!");
-        else
+        } else {
             MESSAGE("But it failed to spit up a thing!");
+        }
 
         ANIMATION(ANIM_TYPE_MOVE, MOVE_STOCKPILE, player);
         MESSAGE("Wobbuffet stockpiled 1!");
 
         ANIMATION(ANIM_TYPE_MOVE, move, player);
         if (move == MOVE_SPIT_UP) {
             HP_BAR(opponent);
-        }
-        else {
+        } else {
```

Comment:
this probably should not be in the PR?

Reply:
It has to be there, because the changed macro requires the {} parentheses.

Reply:
ah, in that case we can probably do the old `do..while(0);` to make it work inside a single line `if`.